src/_lib/modules/charts/tooltip/tooltip.directive.ts

```diff
@@ -0,0 +1,42 @@
+import {Directive, ElementRef, Input, OnChanges, OnInit, Renderer2, SimpleChanges} from '@angular/core';
+
+@Directive({
+    selector: '[nwChartTooltip]'
+})
+export class TooltipDirective implements OnChanges, OnInit {
```

Comment:
Given that we may have a generic tooltip directive in the future I'd recommend calling the class `ChartTooltipDirective` - this also matches the selector naming convention